compareSVsTSVs

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env scala
```

Comment:
You don't have to review this. This is only for comparing to result directories, to check for changes in the output.

Reply:
see commit "Added compareSvsTSVs. Added start of svs-TSV column descriptions do t "

Reply:
Note sure about what you mean by '"co..." <- no good option in our opinion'. Are you referring to the script-name? Or are you referring to the commit-description? (indeed a bit "unlucky" if truncated in Github).